es74x-cloud/src/main/scala/tech/beshu/ror/proxy/es/clients/HighLevelClientBasedIndicesAdminClient.scala

```diff
@@ -0,0 +1,270 @@
+
+  override def exists(request: IndicesExistsRequest, listener: ActionListener[IndicesExistsResponse]): Unit = ???
+
+  override def prepareExists(indices: String*): IndicesExistsRequestBuilder = ???
+
+  override def typesExists(request: TypesExistsRequest): ActionFuture[TypesExistsResponse] = ???
+
+  override def typesExists(request: TypesExistsRequest, listener: ActionListener[TypesExistsResponse]): Unit = ???
+
+  override def prepareTypesExists(index: String*): TypesExistsRequestBuilder = ???
+
+  override def stats(request: IndicesStatsRequest): ActionFuture[IndicesStatsResponse] = ???
+
+  override def stats(request: IndicesStatsRequest, listener: ActionListener[IndicesStatsResponse]): Unit = {
+    esClient
+      .stats(request)
+      .runAsync(handleResultUsing(listener))
+  }
+
+  override def prepareStats(indices: String*): IndicesStatsRequestBuilder = ???
+
+  override def recoveries(request: RecoveryRequest): ActionFuture[RecoveryResponse] = ???
+
+  override def recoveries(request: RecoveryRequest, listener: ActionListener[RecoveryResponse]): Unit = ???
+
+  override def prepareRecoveries(indices: String*): RecoveryRequestBuilder = ???
+
+  override def segments(request: IndicesSegmentsRequest): ActionFuture[IndicesSegmentResponse] = ???
+
+  override def segments(request: IndicesSegmentsRequest, listener: ActionListener[IndicesSegmentResponse]): Unit = ???
+
+  override def prepareSegments(indices: String*): IndicesSegmentsRequestBuilder = ???
+
+  override def shardStores(request: IndicesShardStoresRequest): ActionFuture[IndicesShardStoresResponse] = ???
+
+  override def shardStores(request: IndicesShardStoresRequest, listener: ActionListener[IndicesShardStoresResponse]): Unit = ???
+
+  override def prepareShardStores(indices: String*): IndicesShardStoreRequestBuilder = ???
+
+  override def create(request: CreateIndexRequest): ActionFuture[CreateIndexResponse] = ???
+
+  override def create(request: CreateIndexRequest, listener: ActionListener[CreateIndexResponse]): Unit = ???
+
+  override def prepareCreate(index: String): CreateIndexRequestBuilder = ???
+
+  override def delete(request: DeleteIndexRequest): ActionFuture[AcknowledgedResponse] = ???
+
+  override def delete(request: DeleteIndexRequest, listener: ActionListener[AcknowledgedResponse]): Unit = ???
+
+  override def prepareDelete(indices: String*): DeleteIndexRequestBuilder = ???
+
+  override def close(request: CloseIndexRequest): ActionFuture[CloseIndexResponse] = ???
+
+  override def close(request: CloseIndexRequest, listener: ActionListener[CloseIndexResponse]): Unit = ???
+
+  override def prepareClose(indices: String*): CloseIndexRequestBuilder = ???
+
+  override def open(request: OpenIndexRequest): ActionFuture[OpenIndexResponse] = ???
+
+  override def open(request: OpenIndexRequest, listener: ActionListener[OpenIndexResponse]): Unit = ???
+
+  override def prepareOpen(indices: String*): OpenIndexRequestBuilder = ???
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Usage of ??? operator](https://app.codacy.com/app/scarduzio/elasticsearch-readonlyrest-plugin/pullRequest?prid=4591272)